test/compliance/generate_fbank_data.py

```diff
@@ -0,0 +1,149 @@
+import os
```

Comment:
If you wanted to go above and beyond you'd add argparser and parameterize the path to the executable etc.

This way, locally, someone can generate their own files for testing.